From: =?utf-8?q?S=C3=A9bastien_Delafond?= <sdelafond@gmail.com>
Date: Wed, 11 Dec 2019 13:51:16 -0300
Subject: Use Debian provided /etc/ssl/certs/ca-certificates.cr

Origin: https://salsa.debian.org/debian/python-certifi/-/blob/debian/master/debian/patches/0001-Use-Debian-provided-etc-ssl-certs-ca-certificates.cr.patch
Forwarded: not-necessary
---
 src/pip/_vendor/certifi/core.py | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/src/pip/_vendor/certifi/core.py b/src/pip/_vendor/certifi/core.py
index c3e5466..5c67600 100644
--- a/src/pip/_vendor/certifi/core.py
+++ b/src/pip/_vendor/certifi/core.py
@@ -6,6 +6,7 @@ This module returns the installation location of cacert.pem or its contents.
 """
 import sys
 
+DEBIAN_CA_CERTS_PATH = '/etc/ssl/certs/ca-certificates.crt'
 
 if sys.version_info >= (3, 11):
 
@@ -106,3 +107,13 @@ else:
 
     def contents() -> str:
         return read_text("pip._vendor.certifi", "cacert.pem", encoding="ascii")
+
+
+# Debian: Use system CA certs:
+def where() -> str:
+    return DEBIAN_CA_CERTS_PATH
+
+
+def contents() -> str:
+    with open(where(), "r", encoding="ascii") as data:
+        return data.read()
